demos/flink/flink-1.10/run_flink_on_occlum_glibc.sh

```diff
@@ -0,0 +1,153 @@
+    cp $occlum_glibc/libm.so.6 image/$occlum_glibc
+    cp $occlum_glibc/libnss_files.so.2 image/$occlum_glibc
+    cp -rf ../flink-1.10.1/* image/bin/
+    cp -rf ../hosts image/etc/
+    # Copy Fish and busybox file into Occlum instance
+    mkdir -p image/usr/bin
+    cp ../fish-shell/build/fish image/usr/bin
+    cp ../busybox/busybox image/usr/bin
+    pushd image/bin
+    ln -s /usr/bin/busybox cat
+    ln -s /usr/bin/busybox echo
+    ln -s /usr/bin/busybox awk
+    popd
+    # Build Occlum instance
+    occlum build
+}
+
+run_jobmanager() {
+    # Run Flink JobManager
+    init_flink_instance jobmanager
+    build_flink
+    echo -e "${BLUE}occlum run Flink JobManager${NC}"
+    echo -e "${BLUE}logfile=$log${NC}"
+    occlum run /usr/lib/jvm/java-11-openjdk-amd64/bin/java \
+        -Xmx800m -XX:-UseCompressedOops -XX:MaxMetaspaceSize=256m -XX:ActiveProcessorCount=2 \
+        -Dos.name=Linux \
+        -Dlog.file=$log \
+        -Dlog4j.configuration=file:/bin/conf/log4j.properties \
+        -Dlog4j.configurationFile=file:/bin/conf/log4j.properties \
+        -Dlogback.configurationFile=file:/bin/conf/logback.xml \
+        -classpath /bin/lib/flink-table-blink_2.11-1.10.1.jar:/bin/lib/flink-table_2.11-1.10.1.jar:/bin/lib/log4j-1.2.17.jar:/bin/lib/slf4j-log4j12-1.7.15.jar:/bin/lib/flink-dist_2.11-1.10.1.jar org.apache.flink.runtime.entrypoint.StandaloneSessionClusterEntrypoint \
+        --configDir /bin/conf \
+        --executionMode cluster \
+    &
+}
+
+run_taskmanager() {
+  # Run Flink TaskManager
+    init_flink_instance taskmanager
+    build_flink
+    echo -e "${BLUE}occlum run Flink TaskManager${NC}"
+    echo -e "${BLUE}logfile=$log${NC}"
+    occlum run /usr/lib/jvm/java-11-openjdk-amd64/bin/java \
+        -Xmx800m -XX:-UseCompressedOops -XX:MaxMetaspaceSize=256m -XX:ActiveProcessorCount=2 \
+        -Dos.name=Linux \
+        -Dlog.file=$log \
+        -Dlog4j.configuration=file:/bin/conf/log4j.properties \
+        -Dlog4j.configurationFile=file:/bin/conf/log4j.properties \
+        -Dlogback.configurationFile=file:/bin/conf/logback.xml \
+        -classpath /bin/lib/flink-table-blink_2.11-1.10.1.jar:/bin/lib/flink-table_2.11-1.10.1.jar:/bin/lib/log4j-1.2.17.jar:/bin/lib/slf4j-log4j12-1.7.15.jar:/bin/lib/flink-dist_2.11-1.10.1.jar org.apache.flink.runtime.taskexecutor.TaskManagerRunner \
+        --configDir /bin/conf \
+        -D taskmanager.memory.network.max=64mb \
+        -D taskmanager.memory.network.min=64mb \
+        -D taskmanager.memory.managed.size=128mb \
+        -D taskmanager.cpu.cores=1.0 \
+        -D taskmanager.memory.task.heap.size=256mb \
+    &
+}
+
+run_task() {
+    # Run Flink task
+    init_flink_and_fish_instance task
+    build_flink_and_fish
+    echo -e "${BLUE}occlum run Flink task${NC}"
+    echo -e "${BLUE}logfile=$log${NC}"
+    occlum run /bin/bin/flink run /bin/examples/streaming/WordCount.jar
+}
+
+id=$([ -f "$pid" ] && echo $(wc -l < "$pid") || echo "0")
+
+arg=$1
+case "$arg" in
```

Comment:
If adding one more parameter "environment (in occlum or native)" to help user select how to run the flink modules, it would be better. For example, someone only wants to run tm in occlum, then the use could run tm with parameter "occlum" and run jm and task with parameter "native".